### DISCUSSION TITLE
Got Approved but not validated to Github Student

### DISCUSSION THREAD

**Discussion by gojohnny69** [community/community#56949] [GitHub Education] [answered]
### Hi everyone,

I was approved the student benefits but NO E-mail sent to me. Another Infromation: Hi, ***! You were last verified as a student on yesterday. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

I wait all day without any reply.
I would like to know how long I need to wait after "approved". Will I receive an email from Github? Will it be sent to my main email address, my backup email address or still my application email address？
Repeat: I ask a question if it has been Approved rather than rejected。

**Comment by 10corp** [comment, marked as answer]
I'm in the same boat and it seems that you have to let at least 72 hours pass before the benefits get activated. It also seems like you will get an email when your benefits finally activate, so just be patient.

**Comment by yeetaaron1** [comment]
Its been Ten days for mine, I submitted app and just realized i did it was approved but its been ten days

**Reply by AlessandroSemeraro** [reply]
same boat
